app/src/main/java/org/sopt/korailtalk/core/util/ModifierExt.kt

```diff
@@ -0,0 +1,38 @@
+package org.sopt.korailtalk.core.util
+
+import androidx.compose.foundation.gestures.detectTapGestures
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableLongStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.composed
+import androidx.compose.ui.input.pointer.pointerInput
+
+inline fun Modifier.pressedClickable(
+    crossinline changePressed: (Boolean) -> Unit,
```

Comment:
피그마에 pressed 상태의 디자인이 따로 정의되어 있다면, 여기에 우선 기본값을 두는 건 어떨까요?
```suggestion
    crossinline changePressed: (Boolean) -> Unit = {}
```